src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/FooterBanner.tsx

```diff
@@ -1,17 +1,22 @@
-import { Banner } from "@artsy/palette"
-import { Media } from "Utils/Responsive"
 import { RouterLink } from "System/Router/RouterLink"
+import { FullBleedBanner } from "Components/FullBleedBanner"
+import { useWindowSize } from "Utils/Hooks/useWindowSize"
+import { breakpoints } from "@artsy/palette"
 
 export const FooterBanner: React.FC = () => {
+  const { width } = useWindowSize()
+
   return (
-    <Media greaterThan="xs">
-      <Banner dismissable={false} variant="defaultLight" mx={[-2, -4]}>
-        Gallery or art dealer?
-        <RouterLink to="https://partners.artsy.net">
-          Become a partner
-        </RouterLink>
-        to access the world’s largest online marketplace.
-      </Banner>
-    </Media>
+    <>
+      {width > parseInt(breakpoints.sm, 10) && (
```

Comment:
Using breakpoints because `<Media />` component breaks the banner

Reply:
We should display the banner only on the web view

Reply:
Hmm, have you found out why this doesn't work with `Media`?

Reply:
So: I'm assuming it 'breaks' it because of the `FullBleed` wrapper. Though: it's probably fixable and this isn't at all recommended as the way around it. You could do a few things: very simply use responsive props `display={["none", "block"]}` or use the `className` render props `{className => ...}` to merge with the `FullBleed` children.

Reply:
The use of `useWindowSize` here is going to be breaking server-side rendering, for one.

Reply:
Thanks for raising this @dzucconi - I addressed you comment [here](https://github.com/artsy/force/pull/12120)